lang/csharp/CODECOVERAGE.md

```diff
@@ -0,0 +1,8 @@
+# C# Avro Code Coverage
```

Comment:
misses Apache license header

Reply:
@KyleSchoonover It seems you resolved several comments from us without actually addressing them.

Reply:
None of the other MD files have it. Are we sure we want to add it here?